deployments/warp_routes/weETHs/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Super Symbiotic LRT",
```

Comment:
is this the right name for weETH

Reply:
yes https://discord.com/channels/935678348330434570/1293951457577664582/1303374426074517625

Reply:
https://etherscan.io/address/0x917cee801a67f933f2e6b33fc0cd1ed2d5909d88#readContract#F7